From 5f5c23c3aad20012354c4d2d3802ca18aba7c975 Mon Sep 17 00:00:00 2001
From: Gaurav Vohra <gauravx.vohra@intel.com>
Date: Sun, 21 Jul 2019 23:16:01 +0800
Subject: [PATCH] ASoC: Intel: Common: TLV320AIC3107 Addition of ACPI IDs for
 Machine Driver loading

Adding ACPI IDs for BroadMarket TI CODEC slave/master machine drivers dynamic
loading. Additions were made to soc-acpi-intel-bxt-match.c and
covers the following codecs:

	1. aic3107M    - INT345B
	2. aic3107S    - INT345C

Signed-off-by: Gaurav Vohra <gauravx.vohra@intel.com>
---
 sound/soc/intel/common/soc-acpi-intel-bxt-match.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/sound/soc/intel/common/soc-acpi-intel-bxt-match.c b/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
index 4ceb348..267f7ee 100644
--- a/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
+++ b/sound/soc/intel/common/soc-acpi-intel-bxt-match.c
@@ -39,6 +39,16 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_bxt_machines[] = {
 #endif
 	},
 	{
+		.id = "INT345B",
+		.drv_name = "lhcrb_aic3107M_i2s",
+		.fw_filename = "dsp_fw_release.bin",
+	},
+	{
+		.id = "INT345C",
+		.drv_name = "lhcrb_aic3107S_i2s",
+		.fw_filename = "dsp_fw_release.bin",
+	},
+	{
 		.id = "DLGS7219",
 		.drv_name = "bxt_da7219_max98357a",
 		.fw_filename = "intel/dsp_fw_bxtn.bin",
-- 
1.9.1

